src/objects/zcl_abapgit_object_tabl.clas.abap

```diff
@@ -589,27 +590,31 @@ CLASS ZCL_ABAPGIT_OBJECT_TABL IMPLEMENTATION.
     IF delete_idoc_segment( ) = abap_false.
 
       lv_no_ask = abap_true.
-      SELECT SINGLE tabclass FROM dd02l INTO lv_tabclass
+      SELECT SINGLE tabname tabclass sqltab FROM dd02l
+        INTO CORRESPONDING FIELDS OF ls_dd02l
         WHERE tabname = ms_item-obj_name
         AND as4local = 'A'
         AND as4vers = '0000'.
-      IF sy-subrc = 0 AND lv_tabclass = 'TRANSP'.
+      IF sy-subrc = 0.
```

Comment:
if `AND lv_tabclass = 'TRANSP'` we can avoid calling the function module for structures and views?

Reply:
TRANSP or CLUSTER or POOL

Reply:
updated